dojo/tools/nuclei/parser.py

```diff
@@ -24,12 +24,12 @@ def get_findings(self, filename, test):
         else:
             dupes = {}
             for item in data:
-                template_id = item.get('templateID')
+                template_id = item.get('template-id')
                 info = item.get('info')
```

Comment:
This looks like a format change. Can you make both formats acceptable to maintain backward compatible? It would be best to keep the previous test scan as well as adding the new one to ensure that both formats continue to be supported.

Reply:
Yes, I can.